core/src/main/java/com/pholser/junit/quickcheck/runner/PropertyFalsified.java

```diff
@@ -37,14 +37,16 @@ static AssertionError counterexampleFound(
         Object[] args,
```

Comment:
I'd prefer you not to declare locals as `final`.

Reply:
sure, but would you mind giving some details as to why you prefer this?

Reply:
Certainly! I find that `final`-izing local vars and method parameters introduces noise into the code, and provides a benefit that I don't find very helpful (assurance that the values in question won't change).

When coming into another code base that opts for marking these vars `final`, I'll gladly oblige. In junit-quickcheck and other personal projects of mine, I lean toward avoiding these declarations. I would likely lean on static analysis tools like FindBugs or Checkstyle to warn me of potentially confusing constructs where a local is getting modified a lot.